grails-app/views/email/_approvalsAlert.gsp

```diff
@@ -0,0 +1,53 @@
+<%@ page contentType="text/html"%>
+<style>
+label {
+    font-weight: bold;
+}
+</style>
+<div>
+    <div class="header">
+        <g:render template="/email/header"/>
+    </div>
+    <div>
+        <g:message code='email.requestReceived.message' args='[requisition.destination, requisition.requestedBy, redirectUrl]' />
+    </div>
+    &nbsp;
+</div>
+<div>
+    <table>
+        <thead style="text-align: left;">
```

Comment:
wouldn't `class="text-left"` work? It should be available in bootstrap

Reply:
it doesn't work here